### PR TITLE
Add policy report results to dashboard view

### LINKED ISSUE #798: [Enhancement] Add PolicyReports to the Kubewarden Dashboard

### PR DESCRIPTION
**Pull request by jordojordo**
Fix #798

This adds policy report results to the dashboard view.

Along with the reports added, the layout of the cards has been updated.
- The Policy Servers card has been moved to the end
- Both policy types no longer show the ConsumptionGuage component showing their status
- Events gauge is now displayed in place of the policy status gauge
- The Create buttons have been moved into the header of the card

dashboard

### CODE REVIEW

**Comment by kravciak** [comment]
Functionality-wise I did not find any problem. I have few suggestions / notes about my initial confusion :)

- It's not obvious what Success/Fail/Error mean - should I be scared of red progress bar?
- Rename `Events` to `Compliance` / `Reports` /...?
- Clickable Pass/Fail/Error events so I can take closer look (link to Policy Reporter?)
- Align policy servers progress bar to bottom
- Different `Create ..` button color if any resource already exists - feature?
- Unify colors:
  - Policy Servers: blue & yellow & red & grey
   proposal: change ok  from `blue` to  `green` to match policies
  - Policies: green & red & orange
   proposal: fail = yellow, error=red (fail is just a warning in this case, right?)
   what does error mean - how can I generate error events?

File: pkg/kubewarden/l10n/en-us.yaml [hunk 1]
```diff
@@ -30,23 +30,25 @@ kubewarden:
...
-        description: Kubewarden uses Policy Servers to receive requests to be validated. It does that by executing Kubewarden's policies.
+        description: Policy Servers validate requests
         linkText: Create Policy Server
-        slotTitle: Policy Server Pods
       admissionPolicy:
-        title: Admission Policies
-        description: An Admission Policy is a namespace-wide resource. The policy will process only the requests that are targeting the Namespace where the Admission Policy is defined.
-        linkText: Create Admission Policy
-        slotTitle: Policies
+        title: Namespace Policies
```
Comment (kravciak) [inline pkg/kubewarden/l10n/en-us.yaml:36]: Not sure if this it typo or correct :)
```suggestion
        title: Namespaced Policies
```

File: pkg/kubewarden/l10n/en-us.yaml [hunk 2]
```diff
@@ -30,23 +30,25 @@ kubewarden:
...
+        description: Namespace-wide policy validation
+        linkText: Create Namespace Policy
```
Comment (kravciak) [inline pkg/kubewarden/l10n/en-us.yaml:38]: Same here
```suggestion
        linkText: Create Namespaced Policy
```

**Review by aalves08 — APPROVED** [review]
Codewise, LGTM

**Comment by jordojordo** [comment]
Holding off on this until the UX team can take a look.

**Comment by jordojordo** [comment]
@kravciak I've updated the reports to fix those issues you mentioned after a discussion with the UX team.

dashboard

**Review by aalves08 — APPROVED** [review]
Codewise LGTM

**Comment by kravciak** [comment]
It works fine, just one note - could we show empty bar even if no reports are found?

Screenshot from 2024-11-08 14-28-09

**Comment by jordojordo** [comment]
Here's the updated version with an empty bar:

dashboard

**Comment by kravciak** [comment]
Empty bar is black, is that by intention?

Screenshot from 2024-11-08 16-05-16

**Comment by jordojordo** [comment]
Forgot to check how the bar looked in light mode, should be the correct color now.

**Review by kravciak — APPROVED** [review]

**Pull request merged**